tools/rpkg/R/Connection.R

```diff
@@ -1,13 +1,3 @@
-#' @title DuckDB Connection
-#'
-#' @description
-#' TBD
-#'
-#' @param duckdb_driver FIXME
-#' @param debug FIXME
-#'
-#' @name duckdb_connection
-#' @export
```

Comment:
Was this intended?

Reply:
the constructor duckdb_connection should not be exported